packages/web/src/components/drag-autoscroller/DragAutoscroller.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useRef, useState } from 'react'
```

Comment:
Wonder if this should go in Stems instead? 🤔

Reply:
I think that makes sense. We can start here though and graduate it if we see it needed elsewhere